app/main/db/queries.py

```diff
@@ -243,12 +238,43 @@ def _build_series_view_query(series_id, filters, sorting_orders):
         sub_query.c.consignment_reference,
     )
 
+    if filters:
+        query = _build_browse_filters(query, sub_query, filters)
+
     if sorting_orders:
         query = _build_sorting_orders(query, sub_query, sorting_orders)
     else:
         query = query.order_by(
-            sub_query.c.transferring_body, sub_query.c.series
+            sub_query.c.transferring_body,
+            sub_query.c.series,
+            sub_query.c.consignment_reference,
```

Comment:
This is just an extra bit of ordering you added whilst working on this area, as opposed to anything to do with filtering, right?

Reply:
I don't see it in original file? It looks like it has just been added. But sorting by consignment ref after body and series seems sensible as a default

Reply:
as discussed it make sense to put consignment reference sorting as default